.gitignore

```diff
@@ -25,3 +25,5 @@ typedef.list
 
 /CMakeSettings.json
 /out/*
+
+scratch_file*
```

Comment:
you sure this is right path, arent those created in the sql folder

Reply:
You are right that they are created in `sql` folder. However, I guess the same usage of scratch files might be implemented in other places in future, thus I suggest to ignore them everywhere.

Reply:
This notation is enough for `gitignore` to ignore the files in any folder.